snapshot_network.sh

```diff
@@ -15,11 +15,6 @@ if [[ -z ${BF_COLLECTOR_PASSWORD+x} ]]; then
     exit 1
 fi
 
-if [[ -z ${BF_ACCESS_TOKEN+x} ]]; then
-    echo "Environment variable BF_ACCESS_TOKEN is not set"
-    exit 1
-fi
-
 if [[ -z ${2+x} ]]; then
     echo "Usage: process_collection.sh <inventory file> <batfish settings file> <optional: collection directory>)"
```

Comment:
Comment references `process_collection.sh` but script name is `snapshot_network.sh`

Reply:
old bug. fixed.